integration-tests/test_client_options.py

```diff
@@ -87,3 +88,26 @@ def test_support(insights_client):
     assert "Running command:" in support_result.stdout
     assert "Process output:" in support_result.stdout
     assert "Support information collected in" in support_result.stdout
+
+
+def test_check_show_results(insights_client):
+    """
+    Verify a remediation is returned when checking results from server (--check-results)
+    and then displaying then (--show-results)
+    First I will intorudce a vulnerability to the system and at the
+    end of the test I will set the correct chnod again
```

Comment:
what does "chnod" mean?

Reply:
typo 'chmod'. May be you could write 'file permission' to make it more readable.